src/sentry/runner/commands/devservices.py

```diff
@@ -1,870 +0,0 @@
-from __future__ import annotations
-
-import contextlib
-import functools
-import http
-import json  # noqa
-import os
-import shutil
-import signal
-import subprocess
-import sys
-import time
-import urllib.error
-import urllib.request
-from collections.abc import Callable, Generator
-from concurrent.futures import ThreadPoolExecutor, as_completed
-from typing import TYPE_CHECKING, Any, ContextManager, Literal, NamedTuple, overload
-
-import click
-
-if TYPE_CHECKING:
-    import docker
-
-CI = os.environ.get("CI") is not None
-USE_OLD_DEVSERVICES = os.environ.get("USE_OLD_DEVSERVICES") == "1"
-
-# assigned as a constant so mypy's "unreachable" detection doesn't fail on linux
-# https://github.com/python/mypy/issues/12286
-DARWIN = sys.platform == "darwin"
-
-USE_COLIMA = bool(shutil.which("colima")) and os.environ.get("SENTRY_USE_COLIMA") != "0"
-USE_ORBSTACK = (
-    os.path.exists("/Applications/OrbStack.app") and os.environ.get("SENTRY_USE_ORBSTACK") != "0"
-)
-
-if USE_ORBSTACK:
-    USE_COLIMA = False
-
-if USE_COLIMA:
-    USE_ORBSTACK = False
-
-USE_DOCKER_DESKTOP = not USE_COLIMA and not USE_ORBSTACK
-
-if DARWIN:
-    if USE_COLIMA:
-        RAW_SOCKET_PATH = os.path.expanduser("~/.colima/default/docker.sock")
-    elif USE_ORBSTACK:
-        RAW_SOCKET_PATH = os.path.expanduser("~/.orbstack/run/docker.sock")
-    elif USE_DOCKER_DESKTOP:
-        # /var/run/docker.sock is now gated behind a docker desktop advanced setting
-        RAW_SOCKET_PATH = os.path.expanduser("~/.docker/run/docker.sock")
-else:
-    RAW_SOCKET_PATH = "/var/run/docker.sock"
-
-
-# Simplified from pre-commit @ fb0ccf3546a9cb34ec3692e403270feb6d6033a2
-@functools.cache
-def _gitroot() -> str:
-    from os.path import abspath
-    from subprocess import CalledProcessError, run
-
-    try:
-        proc = run(("git", "rev-parse", "--show-cdup"), check=True, capture_output=True)
-        root = abspath(proc.stdout.decode().strip())
-    except CalledProcessError:
-        raise SystemExit(
-            "git failed. Is it installed, and are you in a Git repository directory?",
-        )
-    return root
-
-
-@contextlib.contextmanager
-def get_docker_client() -> Generator[docker.DockerClient]:
```

Comment:
there's a few imports you'll have to address too

```
josh@CTKP7QRX4L.local ~/d/sentry master $ rg -S 'devservices import'                                                                                      0
bin/split-silo-database
8:from sentry.runner.commands.devservices import get_docker_client

src/sentry/testutils/pytest/relay.py
14:from sentry.runner.commands.devservices import get_docker_client

src/sentry/runner/commands/devserver.py
12:from sentry.runner.commands.devservices import get_docker_client
```

Reply:
yeah, I noticed thought this might be a bit easier to do

Reply:
going to punt this until next week or so for now